examples/examples/keccak_no_syscall.rs

```diff
@@ -0,0 +1,43 @@
+use rkyv::vec::ArchivedVec;
+use tiny_keccak::{Hasher, Keccak};
+
+pub fn main() {
+    let times = 10;
+    let raw_preimage: &ArchivedVec<u32> = ceno_rt::read();
+    let preimage: Vec<u8> = raw_preimage.iter().flat_map(|x| x.to_le_bytes()).collect();
+    for i in 0..times {
+        let digest = keccak256(&preimage)
+            .chunks_exact(4)
+            .map(|chunk| u32::from_le_bytes(chunk.try_into().unwrap()))
+            .collect::<Vec<u32>>();
+
+        if i == 0 {
+            log_digest(digest);
+        }
+        // TODO define serializable struct
+        // ceno_rt::commit::<ArchivedVec<u32>, Vec<u32>>(&result);
```

Comment:
or just commit the hash

Reply:
👍 will come back to this design after we finalized how to set public-io via e2e/cli.